setup.py

```diff
@@ -32,6 +32,7 @@
     cmdclass={'build_ext': build_ext},
     version=_VERSION,
     packages=find_packages(),
+    package_data={"": ["py.typed", "*.pyi"]},
```

Comment:
If setuptools>=v69.0.0 this code is not needed.
https://setuptools.pypa.io/en/latest/history.html#id256

But this is only supported in Python >= 3.8.